### DISCUSSION TITLE
Enable/Disable auto-merge option in a Repository via API

### DISCUSSION THREAD

**Discussion by oscard0m** [octokit/octokit.js#2007] [Q&A] [answered]
I noticed there's the option to enable/disable auto-merge at PR level since https://github.com/octokit/webhooks.js/releases/tag/v7.24.0 (once it is enabled at `Repository` level using GitHub's UI), but I did not find the API to enable/disable repositories' feature programmatically.

It is still not available or I simply did not find it?

Thanks

**Comment by gr2m** [comment, marked as answer]
two GraphQL mutations were introduced two days ago

- [enablePullRequestAutoMerge](https://docs.github.com/en/graphql/reference/mutations#enablepullrequestautomerge)
- [disablePullRequestAutoMerge](https://docs.github.com/en/graphql/reference/mutations#dismisspullrequestreview)

It should be able to use these. I don't think there is an equivalent for the REST API endpoints as of today

**Reply by oscard0m** [reply]
But these are to check/uncheck the auto-merge option in an opened PR:
![image](https://user-images.githubusercontent.com/2574275/107158316-8653b180-6989-11eb-9aa0-452c6bbb8569.png)

But not for enabling the option in a repository, right?
![image](https://user-images.githubusercontent.com/2574275/107158290-6f14c400-6989-11eb-86c8-f321216f06c2.png)

**Reply by gr2m** [reply]
Ah yes, you are right! Sorry, I didn't look into this feature yet, I didn't know there was a distinction.

In that case no, I don't think there is a documented API to enable auto-merge on a repository yet
